python/lsst/daf/persistence/posixStorage.py

```diff
@@ -102,20 +85,23 @@ def getMapperClass(root):
             if os.path.exists(os.path.join(basePath, "_parent")):
```

Comment:
`globals = {}` did come over from an old version of Butler. I can't see any reason why paving over the globals keyword in this context is good and tests seem to run fine without it so I'll take it out.